crates/parser/src/ast/resolve.rs

```diff
@@ -514,13 +514,13 @@ impl Resolver {
         }
         let mut docs = String::new();
         for doc in doc.docs.iter() {
-            if let Some(doc) = doc.strip_prefix("//") {
+            if let Some(doc) = doc.strip_prefix("///") {
                 docs.push_str(doc.trim_start_matches('/').trim());
                 docs.push('\n');
             } else {
-                assert!(doc.starts_with("/*"));
+                assert!(doc.starts_with("/**"));
```

Comment:
I think this assert will need to be conditional instead of an actual assert. Previously this could be an assert because comments only came in these two styles.

Reply:
Wow, I am really sorry for this. It should be fine now.

I wanted to add tests... but I did not see unit tests. It would be weird to add some just for this, but let me know if I should add some.